docs/compass/03-tenant-loading.md

```diff
@@ -0,0 +1,51 @@
+# Initial tenant loading
+
+## Overview
+This document describes how Compass handles initial tenant loading.
+
+## Job
+
+In Compass the user can define his own tenants. There is a special `compass-director-tenant-loader` job provided for that operation. User has to define a ConfigMap which has a JSON file containing tenants as a data. 
+
+The ConfigMap name has to be `compass-director-tenant-config` and the file inside has to be named `tenants.json`. Content of JSON file has to be in the following format:
```

Comment:
format of the JSON should be described in one place (README.md), and then just refer between files.